nautobot/dcim/templates/dcim/inc/device_interface_filter.html

```diff
@@ -1,8 +1,10 @@
 {% load helpers %}
+{% load static %}
 
 {% include "components/panel/header_extra_content_table.html" %}
 
 <div class="pull-right col-md-2 noprint">
     <input class="form-control interface-filter" type="text" placeholder="Filter" title="Filter text (regular expressions supported)" style="height: 23px" />
 </div>
+<script src="{% static 'jquery/jquery-3.7.1.min.js' %}"></script>
```

Comment:
It would seem odd that this is the only place in the code we need to explicitly import jquery, but it worked and figured I would get feedback if there was a better way on Monday.